### DISCUSSION TITLE
Cannot make fetch requests upon deployment

### DISCUSSION THREAD

**Discussion by jackcook85** [vercel/community#1905] [Help]
<img width="955" alt="Screen Shot 2023-03-28 at 7 16 17 PM" src="https://user-images.githubusercontent.com/99844850/228387918-01e32c2e-83df-43fb-b3c5-915292061e70.png">
### Summary

I am trying to deploy a simple website that can persist user accounts. I had everything working locally, but when I deployed via Vercel I started getting a strange error whenever I tried to preform any api actions. It is throwing a 404 error. I'm not sure how I'm meant to interpret this in the context of my problem but I am assuming it means it can't make the request because it can't find it? I'm very confused about this issue and any help would be greatly appreciated.

### Additional information

For reference here is a snap shot of where my code is going wrong and the error message being throw in the console.
<img width="958" alt="Screen Shot 2023-03-28 at 7 10 10 PM" src="https://user-images.githubusercontent.com/99844850/228387666-e1eb8f45-32cd-4aeb-854d-64f8b07646c3.png">

**Comment by adrianstamp** [comment]
Did you try an absolute url?
